site/vendor_custom/prism-okaidia-custom.css

```diff
@@ -0,0 +1,129 @@
+/* PrismJS 1.22.0
+https://prismjs.com/download.html#themes=prism-okaidia&languages=markup+css+clike+javascript&plugins=autolinker */
+/**
+ * okaidia theme for JavaScript, CSS and HTML
+ * Loosely based on Monokai textmate theme by http://www.monokai.nl/
+ * @author ocodia
+ */
+
+ code[class*="language-"],
```

Comment:
You have some linting issues in this file too. There is an unnecessary one space indentation for most of the code, but a couple other small issues. Not sure if you want to ignore or broaden your linting task to capture this.

Reply:
Yeah, I think it's probably best to ignore this folder.